tests/unit/test_credentials.py

```diff
@@ -1,49 +0,0 @@
-#
-# Copyright 2023 Splunk Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-import common
-import pytest
-from splunklib import binding
-
-from solnlib import credentials
-
-
-def test_get_session_key(monkeypatch):
```

Comment:
Yes, we could remove it for now, but I think we'll have to later add tests for `credentials.py` as well.